pkg/controller/certificates/issuing/issuing_controller.go

```diff
@@ -281,14 +291,15 @@ func (c *controller) ProcessItem(ctx context.Context, key string) error {
 }
 
 // failIssueCertificate will mark the condition Issuing of this Certificate as failed, and log an appropriate event
-func (c *controller) failIssueCertificate(ctx context.Context, log logr.Logger, crt *cmapi.Certificate, req *cmapi.CertificateRequest) error {
+func (c *controller) failIssueCertificate(ctx context.Context, log logr.Logger, crt *cmapi.Certificate, req *cmapi.CertificateRequest,
+	conType cmapi.CertificateRequestConditionType) error {
```

Comment:
Can we update the method signature to explain what this new argument does? Or alternatively change the argument to be `detail string` and shift the generation of the message up to the call-site with something like `messageForCertificateConditionType(cmapi.CertificateRequestConditionReady)` (which cannot error) maybe?